Geometry/VeryForwardGeometryBuilder/plugins/TotemRPGeometryESModule.cc

```diff
@@ -416,33 +416,34 @@ void TotemRPGeometryESModule::ApplyAlignments(const edm::ESHandle<DDCompactView>
 
 std::unique_ptr<DDCompactView> TotemRPGeometryESModule::produceMeasuredDDCV(const VeryForwardMeasuredGeometryRecord &iRecord)
 {
-    // get the ideal DDCompactView from EventSetup
-    edm::ESHandle<DDCompactView> idealCV;
-    iRecord.getRecord<IdealGeometryRecord>().get("XMLIdealGeometryESSource_CTPPS", idealCV);
-
-    // load alignments
-    edm::ESHandle<RPAlignmentCorrectionsData> alignments;
-    try {
-        iRecord.getRecord<RPMeasuredAlignmentRecord>().get(alignments);
-    } catch (...) {
-        cout<< "Exception in TotemRPGeometryESModule::produceMeasuredDDCV"
-            << " during iRecord.getRecord<RPMeasuredAlignmentRecord>().get(alignments)"<<endl;
-    }
+  // get the ideal DDCompactView from EventSetup
+  edm::ESHandle<DDCompactView> idealCV;
+  iRecord.getRecord<IdealGeometryRecord>().get("XMLIdealGeometryESSource_CTPPS", idealCV);
 
-    if (alignments.isValid()) {
-        if (verbosity){
-            cout << ">> TotemRPGeometryESModule::produceMeasuredDDCV > Measured geometry: "
-                << alignments->GetRPMap().size() << " RP and "
-                << alignments->GetSensorMap().size() << " sensor alignments applied.\n";
-        }
-    } else {
-        if (verbosity)
-            cout << ">> TotemRPGeometryESModule::produceMeasuredDDCV > Measured geometry: No alignments applied.\n";
+  // load alignments
+  edm::ESHandle<RPAlignmentCorrectionsData> alignments;
+  try {
+    iRecord.getRecord<RPMeasuredAlignmentRecord>().get(alignments);
+  } catch (...) {}
```

Comment:
@jan-kaspar - please, do not use catch(...) if indeed, you want to try this rather then asking for the handle validity. Please use proper exception handling: catch cms::Exception only, print a message, re-throw, etc...